build-scripts/deploy-github.js

```diff
@@ -44,8 +44,10 @@ function getReleaseInfo() {
     const tag = process.env.TAG; // eslint-disable-line no-undef
         throw new Error("No TAG environment variable");
     }
-
+    /* eslint "no-warning-comments": [1, { "terms": ["todo", "fixme"] }] */
+    /* eslint-disable*/
     return {
```

Comment:
Ah, scratched my head on that one for a moment :p.  It's because the `if` statement was removed on line 43 which has broken the syntax of the function.